### PR TITLE
[IMP] wowl: loading indicator fadeout

### PR DESCRIPTION
**Pull request by brboi**

### CODE REVIEW

File: addons/wowl/static/src/components/loading_indicator/loading_indicator.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export class LoadingIndicator extends Component<{}, OdooEnv> {
   });
   rpcIds = new Set();
   blockUITimer?: number;
+  debugMode = true;
```
Comment (kebeclibre) [inline addons/wowl/static/src/components/loading_indicator/loading_indicator.ts:23]: sweet, francois and i are putting debug in the env

**Review by kebeclibre — APPROVED** [review]

File: addons/wowl/static/src/components/loading_indicator/loading_indicator.xml [hunk 2]
```diff
@@ -1,10 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <div t-name="wowl.LoadingIndicator" t-att-class="state.isShown ? 'o_loading' : ''" owl="1">
-        <t t-if="state.isShown">
-            Loading (<t t-esc="state.count"/>)
-        </t>
+    <div t-name="wowl.LoadingIndicator" class="o_loading" t-att-class="{ fadeout: !state.isShown }" owl="1">
```
Comment (kebeclibre) [inline addons/wowl/static/src/components/loading_indicator/loading_indicator.xml]: woops, why not a good owl' transition :D ?

**Review by kebeclibre — APPROVED** [review]

File: addons/wowl/static/src/components/loading_indicator/loading_indicator.xml (hunk 2 above)
Comment (brboi) [reply addons/wowl/static/src/components/loading_indicator/loading_indicator.xml]: Good Owl' transitions only helps in one specific situation: when the node is attached/detached from the dom :-)

The loading indicator is always present on the DOM.

Maybe we should change that ? @FrancoisGe , more infos on that please ?

Comment (FrancoisGe) [reply addons/wowl/static/src/components/loading_indicator/loading_indicator.xml]: @brboi How you want to remove the loading indicator when we don't need it ? If you find a good way to do it, you can do it :)

Comment (brboi) [reply addons/wowl/static/src/components/loading_indicator/loading_indicator.xml]: You can check commit V2: this use t-transition by inserting a sub-component. Ugly IMO.

As fge suggested, we could also make the loading indicator a service...

@kebeclibre any thoughts on this ?

File: addons/wowl/static/src/components/loading_indicator/loading_indicator.ts [hunk 3]
```diff
@@ -43,6 +45,9 @@ export class LoadingIndicator extends Component<{}, OdooEnv> {
       clearTimeout(this.blockUITimer);
       this.env.services.ui.unblock();
       this.state.isShown = false;
+      this.env.browser.setTimeout(() => {
+        this.el!.style.display = "none";
+      }, 400);
```
Comment (brboi) [inline addons/wowl/static/src/components/loading_indicator/loading_indicator.ts]: This is tied to the CSS transition timing: U.G.L.Y. !

**Review by aab-odoo — APPROVED** [review]

**Pull request merged**